bip-0009.mediawiki

```diff
@@ -0,0 +1,167 @@
+<pre>
+  BIP: 9
+  Title: Version bits with timeout and delay
+  Author: Pieter Wuille <pieter.wuille@gmail.com>, Peter Todd <pete@petertodd.org>, Greg Maxwell <greg@xiph.org>, Rusty Russell <rusty@rustcorp.com.au>
+  Status: Draft
+  Type: Informational Track
+  Created: 2015-10-04
+</pre>
+
+==Abstract==
+
+This document specifies a proposed change to the semantics of the 'version' field in Bitcoin blocks, allowing multiple backward-compatible changes (further called called "soft forks") being deployed in parallel. It relies on interpreting the version field as a bit vector, where each bit can be used to track an independent change. These are tallied each retarget period. Once the consensus change succeeds or times out, there is a "fallow" pause after which the bit can be reused for later changes.
```

Comment:
฿tcDrak <notifications@github.com> writes:
>> @@ -0,0 +1,167 @@
>> +<pre>
>> +  BIP: 9
>> +  Title: Version bits with timeout and delay
>> +  Author: Pieter Wuille <pieter.wuille@gmail.com>, Peter Todd <pete@petertodd.org>, Greg Maxwell <greg@xiph.org>, Rusty Russell <rusty@rustcorp.com.au>
>> +  Status: Draft
>> +  Type: Informational Track
>> +  Created: 2015-10-04
>> +</pre>
>> +
>> +==Abstract==
>> +
>> +This document specifies a proposed change to the semantics of the 'version' field in Bitcoin blocks, allowing multiple backward-compatible changes (further called called "soft forks") being deployed in parallel. It relies on interpreting the version field as a bit vector, where each bit can be used to track an independent change. These are tallied each retarget period. Once the consensus change succeeds or times out, there is a "fallow" pause after which the bit can be reused for later changes.
>
>> allowing multiple backward-compatible changes (further called called "soft forks") being deployed in parallel
>
> allowing multiple backward-compatible changes (further called called "soft forks") to be deployed in parallel
>
> s/being/to be/

And s/called called/called/ :)

Thanks,
Rusty.